tests/opentitan/module_tests/aes_core/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <verilated_vcd_c.h>
+
+#define VL_DEBUG
+#include "Vaes_core.h"
+#include "verilated.h"
+
+static vluint64_t main_time = 0;
+
+double
+sc_time_stamp()
+{
+  return main_time;
+}
+
+int main (int argc, char **argv) {
+  Verilated::commandArgs(argc, argv);
+  Vaes_core *top = new Vaes_core();
+
+  Verilated::traceEverOn(true);
+  VerilatedVcdC* tfp = new VerilatedVcdC;
+  top->trace(tfp, 99);
+  tfp->open("dump.vcd");
+  
+  while (!Verilated::gotFinish() && (main_time < 100)) {
```

Comment:
We're obviously not aiming for a full-fledged testbench here, but try to at least:

1.  Clock the core
2. Bring it out of reset
3. Eyeball the .vcd if the signals seem ok.